File: CS/Modules.TeamList/TeamList.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Modules.Infrastructure;

namespace Modules.TeamList {
    public class TeamList {
        static TeamList() {
            Teams = Modules.Infrastructure.TeamList.CreateSampleData();
        }
        public static List<Team> Teams;
    }
}

File: VB/Modules.TeamList/TeamList.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports Modules.Infrastructure

Namespace Modules.TeamList
	Public Class TeamList
		Shared Sub New()
			Teams = Modules.Infrastructure.TeamList.CreateSampleData()
		End Sub
		Public Shared Teams As List(Of Team)
	End Class
End Namespace
